src/app/setup.py

```diff
@@ -43,8 +43,8 @@ def find_version(version_file):
         "ruamel.yaml<0.17",
         "stomp.py<6.2.0",
         "tornado<7",
-        "watchdog<1",
-        "yapconf==0.3.7",
+        "watchdog>2.1.0",
+        "yapconf==1.0.0",
```

Comment:
We don't need to be so strict with the version here.  We had it pinned to 0.3.7 before because we discovered 1.0.0 was going to be breaking.  We should loosen this up to be `>=1.0.0` and only pin it in the future if something breaks again.

Reply:
Updated